### DISCUSSION TITLE
Add toggle on the main dashboard to easily switch between private and public server

### DISCUSSION THREAD

**Discussion by EusebiuOprinoiu** [rustdesk/rustdesk#8362] [General]
Hello!

Would you please consider adding a toggle on RustDesk's main dashboard to easily switch between the private server configured under Network and the public one?

I connect primarily to the computers of family members (who all use the private server), but from time to time, I also need to connect to other people to whom I do not want to give permanent access to my server.

Right now, to switch from the private server to the public one, I have to remove the Server ID and the encryption key whenever I want to connect to someone outside my trusted circle of friends and family. (and add them back once the session is over)

A toggle would allow us to change servers at the click of a button without the need to edit our settings:
- Toggle ON: use the private server configured in settings
- Toggle OFF: ignore settings and use the public server

**Comment by rustdesk** [comment]
duplicate: https://github.com/rustdesk/rustdesk/discussions/1258
